web/themes/new_weather_theme/templates/wfo-info/node--wfo-info.html.twig

```diff
@@ -1,41 +1,111 @@
-<h1>
-    WFO Information Page
-</h1>
-
-<h2>Image</h2>
-{{content.field_wfo_image}}
-
-<h2>WFO</h2>
-<dl>
-    {% for key,val in wfo %}
-        <dt>{{key}}</dt>
-        <dd>{{val}}</dd>
-    {% endfor %}
-</dl>
-
-<h2>About</h2>
-{{content.body}}
-
-<h2>Phone number</h2>
-<dl>
-    {% for key,val in phone %}
-        <dt>{{key}}</dt>
-        <dd>{{val}}</dd>
-    {% endfor %}
-</dl>
-
-<h2>Address</h2>
-<dl>
-    {% for key,val in address %}
-        <dt>{{key}}</dt>
-        <dd>{{val}}</dd>
-    {% endfor %}
-</dl>
-
-<h2>External links</h2>
-<dl>
-    {% for key,val in external_links %}
-        <dt>{{key}}</dt>
-        <dd>{{val}}</dd>
-    {% endfor %}
-</dl>
+<div class="wx-wfo-info">
+  {% if content.field_wfo_image is not empty %}
+    <div class="wx-wfo-image grid-container padding-0 tablet:padding-x-2 desktop:padding-x-4">
+      {{ content.field_wfo_image }}
+    </div>
+  {% endif %}
+
+  <div class="grid-container">
+    <h1 class="text-normal text-primary-darker">
```

Comment:
Something I missed, but I think we need a little extra space below the h1 on desktop. Could you add `desktop:margin-bottom-3`

Reply:
👍